build.sbt

```diff
@@ -506,7 +507,7 @@ def shadeSbtSettingsForModule(
       }.value
     },
     shadeOwnNamespaces := Set("bloop"),
-    shadeIgnoredNamespaces := Set("com.google.gson"),
+    shadeIgnoredNamespaces := Set("com.google.gson", "scala"),
```

Comment:
Why is `scala` added here?

Reply:
Due to the `collection-compat` library that is brought in by coursier. It should be binary compatible and has a nice Mima job, so we should be fine, so we cna ignore it.

Reply:
Ok, but `collection-compat` is not the only artifact that use the `scala` namespace. What about the scala library?

Reply:
So that did not pop up as being a problem and it should also be binary compatible for any major version.

Besides, we are already doing it for example here: https://github.com/scalacenter/bloop/blob/master/build.sbt#L394